blocked-edges/4.3.2.yaml

```diff
@@ -1,3 +1,3 @@
-# https://bugzilla.redhat.com/show_bug.cgi?id=1802248, https://bugzilla.redhat.com/show_bug.cgi?id=1805444, https://bugzilla.redhat.com/show_bug.cgi?id=1808429
 to: 4.3.2
 from: .*
+# 4.2 -> 4.3 updates hit TargetDown for cluster-autoscaler-operator, fixed in 4.3.5, https://bugzilla.redhat.com/show_bug.cgi?id=1801300
```

Comment:
Not clear here why we are blocking updates from prior 4.3.z

Reply:
https://github.com/openshift/cincinnati-graph-data/pull/119#discussion_r395336806